### ISSUE TITLE
vz: unsupported build target macOS version for 13.0

### ISSUE THREAD

**Issue by swalkinshaw** [lima-vm/lima#1182]
### Description

macOS version: 13.0.1
Template: default [vz.yml](https://github.com/lima-vm/lima/blob/master/examples/experimental/vz.yaml)

I'm trying out VZ support and my `ha.stderr.log` log looks like this:

```plain
{"level":"debug","msg":"Creating iso file /Users/me/.lima/example-com/cidata.iso","time":"2022-11-19T15:13:35-05:00"}
{"level":"debug","msg":"Using /var/folders/s8/0tthc8jj04zd_4xzr8hwl1y40000gn/T/diskfs_iso734331136 as workspace","time":"2022-11-19T15:13:35-05:00"}
{"level":"debug","msg":"OpenSSH version 9.0.1 detected","time":"2022-11-19T15:13:35-05:00"}
{"level":"debug","msg":"AES accelerator seems available, prioritizing aes128-gcm@openssh.com and aes256-gcm@openssh.com","time":"2022-11-19T15:13:35-05:00"}
{"level":"info","msg":"Starting VZ (hint: to watch the boot progress, see \"/Users/me/.lima/example-com/serial.log\")","time":"2022-11-19T15:13:35-05:00"}
{"level":"debug","msg":"Start udp server listening on: 127.0.0.1:57460","time":"2022-11-19T15:13:35-05:00"}
{"level":"debug","msg":"Start tcp server listening on: 127.0.0.1:58956","time":"2022-11-19T15:13:35-05:00"}
{"level":"fatal","msg":"unsupported build target macOS version for 13.0","time":"2022-11-19T15:13:35-05:00"}
```

Looks like `vz` is returning that error here:
https://github.com/Code-Hex/vz/blob/9de1c1041d5e560f46f9f94fc499a1cc72fd92e5/osversion.go#L108

Lima only handles the other kind of OS error:
https://github.com/lima-vm/lima/blob/cd504a41491faa3e05ae82fc9bade406c0e3684e/pkg/vz/vz_driver_darwin.go#L52

Which means the error from vz is being logged directly without any additional context. I'm _assuming_ there's some API that's not supported/available but I have no idea which (or why).

**Comment by kdrag0n** [comment]
I had the same issue and my theory is that the virtualization entitlement isn't taking effect because of code signing. Building from source worked for me, persumably because it gets ad-hoc signing. This might be specific to Apple Silicon.

**Comment by AkihiroSuda** [comment]
The binaries are built on macOS 12, that might be the reason https://github.com/lima-vm/lima/blob/cd504a41491faa3e05ae82fc9bade406c0e3684e/.github/workflows/release.yml#L15

**Comment by swalkinshaw** [comment]
Confirmed that building it from source fixed it.

**Comment by balajiv113** [comment]
@AkihiroSuda 
Should we compile from macOS13 ? 

Github action doesn't support macOS 13 as of now. It might take some time for sure. 
If needed, we could look into cirrus-ci to build release artifacts for macOS

**Comment by AkihiroSuda** [comment]
@balajiv113 
I assume most users use Homebrew, so I guess we do not need to upload macOS 13 binaries by ourselves

**Comment by AkihiroSuda** [comment]
@Code-Hex Thanks, trying in:
- https://github.com/lima-vm/lima/pull/1196

**Comment by AkihiroSuda** [comment]
Solved in v0.14.0-beta.1
- https://github.com/lima-vm/lima/releases/tag/v0.14.0-beta.1
